### PR TITLE
Adjust vega by appropriate volatility.

### PR DESCRIPTION
**Pull request by daminuk**
* Multiply the vega sensitivites by the appropriate volatility amount.

### CODE REVIEW

File: modules/pricer/src/test/java/com/opengamma/strata/pricer/swaption/SwaptionNormalVolatilityDataSets.java [hunk 1]
```diff
@@ -61,10 +66,18 @@
       IborRateSwapLegConvention.of(USD_LIBOR_3M);
   public static final FixedIborSwapConvention USD_1Y_LIBOR3M =
       ImmutableFixedIborSwapConvention.of("USD-Swap", USD_FIXED_1Y_30U360, USD_IBOR_LIBOR3M);
+  private static final List<ParameterMetadata> PARAMETER_METADATA =
+      MapStream.zip(TIMES.stream().boxed(), TENORS.stream().boxed())
+      .map(SwaptionSurfaceExpiryTenorParameterMetadata::of)
```
Comment (jodastephen) [inline modules/pricer/src/test/java/com/opengamma/strata/pricer/swaption/SwaptionNormalVolatilityDataSets.java:71]: Indentation doesn't look right here

File: modules/measure/src/test/java/com/opengamma/strata/measure/swaption/SwaptionTradeCalculationsTest.java [hunk 2]
```diff
@@ -64,13 +68,30 @@ public void test_pv01() {
     CurrencyParameterSensitivities expectedPv01CalBucketed = pvParamSens.multipliedBy(1e-4);
     PointSensitivities pvVegaPointSens = pricer.presentValueSensitivityModelParamsVolatility(RTRADE, provider, VOLS);
     CurrencyParameterSensitivities expectedPv01VegaCal = VOLS.parameterSensitivity(pvVegaPointSens);
+    CurrencyParameterSensitivities expectedPv01VegaCalAdjusted = expectedPv01VegaCal.getSensitivities().stream()
```
Comment (jodastephen) [inline modules/measure/src/test/java/com/opengamma/strata/measure/swaption/SwaptionTradeCalculationsTest.java:71]: Is there no way to use `CurrencyParameterSensitivities.toBuilder().filterSensitivity(...) ` followed by `CurrencyParameterSensitivities.mapSensitivities(...)` ? If not, then maybe Strata needs an enhancement

Comment (daminuk) [reply modules/measure/src/test/java/com/opengamma/strata/measure/swaption/SwaptionTradeCalculationsTest.java]: I've added a `mapSensitivities(BiFunction<ParameterMetadata, Double, Double>)` method to the builder.

**Review by jodastephen — APPROVED** [review]
Was going to suggest using `ObjDoubleToDoubleFunction` but on closer look it wouldn't really help as the class is boxed anyway.

**Pull request merged**